packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -234,6 +244,10 @@ export const Popover = React.forwardRef((props, forwardedRef) => {
     placement = 'bottom-start',
     onVisibleChange,
     closeOnOutsideClick = true,
+    middleware,
+    //
+    // extra props
+    positionReference,
     //
     // dom props
```

Comment:
Should this be called "public props" to stay consistent with `PopoverPublicProps`

Reply:
no, those are dom props, as in they are passed to the underlying div